l10n_th_tin_service/views/res_partner_view.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_partner_form_move_vat" model="ir.ui.view">
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <field name="name_company" position="after">
+                <group>
+                    <field
+                        name="vat"
+                        class="oe_edit_only"
+                        attrs="{'invisible': [('is_company', '=', False)], 'required': [('is_company', '=', True)]}"
+                    />
```

Comment:
the `placeholder` should be updated from `e.g. BE0477472701` to something like `e.g. 0105519005906`.
```suggestion
                    <field
                        name="vat"
                        class="oe_edit_only"
                        placeholder="e.g. 0105519005906"
                        attrs="{'invisible': [('is_company', '=', False)], 'required': [('is_company', '=', True)]}"
                    />
```